chapters/usage-profile.md

```diff
@@ -0,0 +1,326 @@
+
+
+### 31.4.2 Intent
+
+Description of the package release date for strict identification of the prerequisite assumptions of usage scope by the package supplier.
+
+### 31.4.3 Examples
+
+EXAMPLE 1 Tag: `PackageReleaseDate:`
+
+```text
+PackageReleaseDate: 2010-01-29T18:30:22Z
+```
+
+EXAMPLE 2 RDF: Property `spdx:packageReleseDate` in class `spdx:UsageScope`
+
+```text
+<UsageScope rdf:about="...">
+    <packageReleaseDate> 2010-01-29T18:30:22Z </packageReleaseDate>
+</UsageScope>
+```
+
+## 31.5 Package Built Date field <a name="31.5"></a>
+
+### 31.5.1 Description
+
+This field provides a place for recording built date of the package 
+
+Table 31.5 — Metadata for the package built date
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 0..1 |
+| Format | `YYYY-MM-DDThh:mm:ssZ`<br>where:<br><ul><li>`YYYY` is year</li><li>`MM` is month with leading zero</li><li>`DD` is day with leading zero</li><li>`T` is delimiter for time</li><li>`hh` is hours with leading zero in 24 hour time</li><li>`mm` is minutes with leading zero</li><li>`ss` is seconds with leading zero</li><li>`Z` is universal time indicator</li></ul> |
+
+
+### 31.5.2 Intent
+
+Description of the built date of the package for strict identification of the prerequisite assumptions of usage scope by the package supplier.
+It should recorded from build system tools or date field of the package file.
+
+### 31.5.3 Examples
+
+EXAMPLE 1 Tag: `PackageBuiltDate:`
+
+```text
+PackageBuiltDate: 2010-01-29T18:30:22Z
+```
+
+EXAMPLE 2 RDF: Property `spdx:packageBuiltDate` in class `spdx:UsageScope`
+
+```text
+<UsageScope rdf:about="...">
+    <packageBuiltDate> 2010-01-29T18:30:22Z </packageBuiltDate>
+</UsageScope>
+```
+
+## 31.6 Valid Until Date field <a name="31.6"></a>
+
+### 31.6.1 Description
+
+Identify expiration date of designated "usage scope" descriptions which declared by supplier of the package. 
+
+Table 31.6 — Metadata for the valid until date
+
+| Attribute | Value |
+| --------- | ----- |
+| Required | No |
+| Cardinality | 0..1 |
+| Format | `YYYY-MM-DDThh:mm:ssZ`<br>where:<br><ul><li>`YYYY` is year</li><li>`MM` is month with leading zero</li><li>`DD` is day with leading zero</li><li>`T` is delimiter for time</li><li>`hh` is hours with leading zero in 24 hour time</li><li>`mm` is minutes with leading zero</li><li>`ss` is seconds with leading zero</li><li>`Z` is universal time indicator</li></ul> |
+
+
+### 31.6.2 Intent
+
+Description of the expiration date for specific usage scope.
+
+### 31.6.3 Examples
+
+EXAMPLE 1 Tag: `ValidUntilDate:`
+
+```text
+ValidUntilDate: 2030-01-29T18:30:22Z
+```
+
+EXAMPLE 2 RDF: Property `spdx:validUntilDate` in class `spdx:UsageScope`
+
+```text
+<UsageScope rdf:about="...">
+    <validUntilDate> 2030-01-29T18:30:22Z </validUntilDate>
+</UsageScope>
+```
+
+## 31.7 Valid Until Event field <a name="31.7"></a>
```

Comment:
Same question here as for the "Valid Until Date" field above. What does it mean for this to be an "expiration condition," and what it mean for something here to be expiring or become invalid?